docs/release-notes/august-2024.md

```diff
@@ -0,0 +1,128 @@
+- Fixed a regression in which clicking outside of the **Findings** page filter component did not clear all filters.
+- Various copy edits to the Dashboard (beta) page. <!-- 16176 -->
+- Fixed an issue in which untriaged findings could be marked as reopened when creating Jira tickets from the **Finding details** page. <!-- 15969 -->
+- Fixed a bug in which the **Dashboard** did not display the correct number of findings. <!-- 15935-->
+
+## 💻 Semgrep Code
+
+### Added
+
+- **Docker**: Semgrep ellipses `...` are now allowed in patterns for `HEALTHCHECK` commands.
+- **Terraform**: Added support for `.tfvars` files. <!-- SAF-1481 -->
+
+### Changed
+
+- Semgrep CLI's `--debug` flag no longer generates profiling information, including time and scan performance measurements. To obtain this information, use `--time`.
+
+### Fixed
+
+- Fixed an error with Julia list comprehensions. For example, the pattern `[$A for $B in $C]` matches `[x for y in z]` and result in three bindings `[$A/x,$B/y,$C/z]` instead of one `[$A/x]`.
+- Fixed an issue resulting in deadlock when a scan has interfile analysis and tracing enabled and the number of subprocesses is greater than 1 (`j < 1`). <!-- SAF-1157 -->
+- Fixed an issue where the number of files reported as scanned by Semgrep CLI was inflated due double-counting of generic and regex modes. <!-- SAF-507 -->
+- `--debug` now generates fewer log entries. Additionally, when the number of ignored files, rules, or other entities is too large, Semgrep indicates this in the logs with `<SKIPPED DATA>` to keep the output minimal.
+
+## ⛓️ Semgrep Supply Chain
+
+### Added
+
+- You can now filter and view EPSS scores for your Supply Chain findings.
+
+### Changed
+
+- The link to the Supply Chain findings page in Semgrep AppSec Platform filters to the specific repository and `ref` on which the findings are detected.
+
+### Fixed
+
+- Fixed an issue where Supply Chain's Findings Detail pages weren't showing detailed error information.
+
+## 🤖 Semgrep Assistant
+
+### Added
+
+- Assistant Memories is now in **public beta**. Assistant Memories allows you to tailor Assistant's remediation guidance to your organization's standards and defaults on a per-project, per-rule basis.
+- Added the ability for you to use your own OpenAI API key instead of Semgrep's. This allows you to have complete control over how OpenAI handles your data.
+- Added the ability to query for Assistant's remediation guidance via the [Findings API](https://semgrep.dev/api/v1/docs/#tag/Finding/operation/semgrep_app.core_exp.findings.handlers.issue.openapi_list_recent_issues).
+
+## 🔐 Semgrep Secrets
+
+### Changed
+
+- The Secrets findings page in Semgrep AppSec Platform has been updated to match those for Semgrep Code and Semgrep Supply Chain.
```

Comment:
```suggestion
- The **Secrets** page in Semgrep AppSec Platform has been updated to match those for Semgrep Code and Semgrep Supply Chain.
```

The name of the page (h1) is just Secrets 
![image](https://github.com/user-attachments/assets/bc80befe-088d-41b1-9ea0-5d130003978e)